.daily_canary

```diff
@@ -1,4 +1,4 @@
    ___     ___
-  (- *)   (O o)   |     Y   &  +
```

Comment:
I know it's already unstable, but should we run the multi-threaded CI (`.threading_canary`) a few times over this PR?

Reply:
Sure, I'll run a few. It'll add a red badge to this PR we'll need to ignore, but I guess we can look at the failures and see if they look different.

Reply:
So far, failures look the same - `partitions_test` is failing persistently, `reconfiguration_test_cft` and `governance_test` are failing most of the time.